source/javascripts/core/models/BitriseYmlService.ts

```diff
@@ -50,6 +70,22 @@ function createWorkflow(workflowId: string, yml: BitriseYml, baseWorkflowId?: st
   return copy;
 }
 
+function updateWorkflow(workflowId: string, workflow: WorkflowYmlObject, yml: BitriseYml): BitriseYml {
+  const copy = deepCloneSimpleObject(yml);
+
+  mapValues(workflow, (value: string, key: never) => {
```

Comment:
Why don't we use loadash's `merge` method?

Reply:
Because the lodash merge will not delete the key from the YML if it's updated to an empty value (null, undefined or '')